content/rs/release-notes/rs-7-2-4-releases/rs-7-2-4-tba.md

```diff
@@ -0,0 +1,152 @@
+---
+Title: Redis Enterprise Software release notes 7.2.4-TBA (September 2023)
+linkTitle: 7.2.4-TBA (September 2023)
+description: Improved cluster recovery with manually uploaded modules. Support package contains supervisorctl status. Configure port range using rladmin and REST API. License API returns the number of used shards (RAM & flash).
+compatibleOSSVersion: Redis 7.2.0
+weight: 71
+alwaysopen: false
+categories: ["RS"]
+aliases: 
+---
+
+This is a maintenance release for ​[​Redis Enterprise Software version 7.2.4](https://redis.com/redis-enterprise-software/download-center/software/).
+
+## Highlights
+
+This version offers:
+
+- Improved cluster recovery with manually uploaded modules
+
+- Support package enhancements
+
+- Configurable port range
+
+- License API enhancements
+
+## New in this release
+
+### Enhancements
+
+- Cluster recovery with manually uploaded modules 
+
+    - For clusters containing databases with manually uploaded modules, [cluster recovery]({{<relref "/rs/clusters/cluster-recovery">}}) is now seamlessly integrated
+
+- Support package now contains `supervisorctl` status when created by the [`rladmin` command]({{<relref "/rs/installing-upgrading/creating-support-package#command-line-method">}}) (RS107879)
+
+- Port range (`reserved_ports`) is now configurable using [`rladmin`]({{<relref "/rs/references/cli-utilities/rladmin/cluster/config">}}) or the [REST API]({{<relref "/rs/references/rest-api/requests/cluster">}})
+
+    - Removed `rlutil reserved_ports`, which was deprecated in Redis Enterprise Software [version 7.2.4-52]({{<relref "/rs/release-notes/rs-7-2-4-releases/rs-7-2-4-52#api-deprecations">}})
+
+- [License REST API requests]({{<relref "/rs/references/rest-api/requests/license">}}) return the number of used shards (RAM & flash)
+
+#### Redis modules
+
+Redis Enterprise Software version 7.2.4-TBA includes the following Redis Stack modules (no changes since [Redis Enterprise Software version 7.2.4-52]({{<relref "/rs/release-notes/rs-7-2-4-releases/rs-7-2-4-52">}}):
+
+- [RediSearch 2.8.4](https://github.com/RediSearch/RediSearch/releases/tag/v2.8.4)
+
+- [RedisJSON 2.6.6](https://github.com/RedisJSON/RedisJSON/releases/tag/v2.6.6)
+
+- [RedisTimeSeries 1.10.4](https://github.com/RedisTimeSeries/RedisTimeSeries/releases/tag/v1.10.4)
+
+- [RedisBloom 2.6.3](https://github.com/RedisBloom/RedisBloom/releases/tag/v2.6.3)
+
+- [RedisGears 2.0.11](https://github.com/RedisGears/RedisGears/releases/tag/v2.0.11-m12)
+
+### Resolved issues
+
+- RS107986 - Allow ports lower than 1024 when configuring audits using `rladmin`
+
+- RS105335 - Add error handling to [`CLIENT NO-TOUCH`](https://redis.io/commands/client-no-touch/) to return an error if the Redis database version is earlier than 7.2.0 or the command is [restricted by ACL rules](https://docs.redis.com/latest/rs/release-notes/rs-7-2-4-releases/rs-7-2-4-52/#command-limitations)
```

Comment:
My intention was that the link to the command limitation section will have text like "it handles this limitation", so that the user knows it links to the description of the limitation (as I suggested in ticket https://redislabs.atlassian.net/browse/DOC-2671)

IMO the current link text is misleading: when a user clicks the link "restricted by ACL rules" it assumes that it links to info about what are ACL rules, not to the previous RN page.

Reply:
Fixed